.github/workflows/push-artifact-flutter.yml

```diff
@@ -5,15 +5,23 @@ on:
     branches:
       - main
     paths:
-      - "sync-todo/v1/generated/flutter_todo/**"
+      - "sync-todo/v2/client/flutter/**"
 
 jobs:
   push:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+
+      - name: copy template-realm.json to app pre-bluehawking
+        uses: canastro/copy-file-action@master
+        source: ./sync-todo/v2/template-realm.json
+        target: ./sync-todo/v2/client/flutter/assets/config/realm.json
```

Comment:
Check how this is done in the bluehawk.sh instead on v1. More consistently if it's done as part of the bluehawk step (unfortunately bluehawk™ doesn't currently have a way to do this, and JSON doesn't allow comments)

Reply:
there was some reason that i did it here instead of the bluehawk step, which now escapes me. 

next week, will revisit this and see if i can figure out what my reasoning is now. if not,, will do similar to the v1 bluehawk.sh script